### DISCUSSION TITLE
UCB CS170: Efficient Algorithms and Intractable Problems - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#116] [Announcements]
# UCB CS170: Efficient Algorithms and Intractable Problems - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84%E4%B8%8E%E7%AE%97%E6%B3%95/CS170/

**Comment by AHandsomePython** [comment]
同时强推MIT 6.006 6.854 6.046

**Reply by sbwcwso** [reply]
赞同，CS170 感觉在课堂效果没有 MIT 的好，课后的教材感觉跟 CLRS 也差好多。 
CS170 学到一半，学不下去了，转去学 MIT 的算法课了，希望能坚持下去。

**Comment by lsy1109** [comment]
如果目标是刷算法题的话，这门课的学习有用吗

**Reply by trace1729** [reply]
或许直接刷题会更直接

**Reply by QihanZhao** [reply]
用处不大

**Reply by dengyunsheng250** [reply]
用处很大

**Reply by Eleven-6** [reply]
用处很大

**Comment by HippoGrasshopper** [comment]
请问有同学有教材里伪代码的模板吗？觉得好好看但是哪儿也找不到/(ㄒoㄒ)/~~

**Comment by CliPg** [comment]
现在hw有开放吗

**Comment by dghtucs** [comment]
680147729创建群聊学习讨论qq

**Comment by icewalk7** [comment]
大家是直接学的教材还是看的recording啊
